cloudinit/bastion.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
@chuckha we still need the bastion stuff right?

Reply:
This is very AWS specific, so we should probably not migrate this here.

Reply:
right, let's leave this out for now. We were just talking about this yesterday and don't have a resolution. Let's leave it out of this provider and rally on a discussion here https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/884